sdgym/synthesizers/generate.py

```diff
@@ -0,0 +1,231 @@
+"""Synthesizers module."""
+
+from sdv.lite import TabularPreset
+from sdv.relational import HMA1
+from sdv.tabular import CTGAN, TVAE, CopulaGAN, GaussianCopula
+from sdv.timeseries import PAR
+
+from sdgym.synthesizers.base import BaselineSynthesizer
+from sdgym.synthesizers.sdv import FastMLPreset, SDVRelationalSynthesizer, SDVTabularSynthesizer
+
+SYNTHESIZER_MAPPING = {
+    'FastMLPreset': TabularPreset,
+    'GaussianCopulaSynthesizer': GaussianCopula,
+    'CTGANSynthesizer': CTGAN,
+    'CopulaGANSynthesizer': CopulaGAN,
+    'TVAESynthesizer': TVAE,
+    'PARSynthesizer': PAR,
+    'HMASynthesizer': HMA1,
+}
+
+
+def create_sdv_synthesizer_variant(synthesizer_class, synthesizer_parameters):
+    """Create a new synthesizer that is a variant of an SDV tabular synthesizer.
+
+    Args:
+        synthesizer_class (string):
+            The name of the SDV synthesizer class. The available options are
+            ('FastMLPreset', 'GaussianCopulaSynthesizer', 'CTGANSynthesizer',
+            'CopulaGANSynthesizer', 'TVAESynthesizer', 'PARSynthesizer', ''HMASynthesizer')
```

Comment:
Can we list them in bullet points instead, something similar to what we have [here](https://github.com/sdv-dev/SDV/blob/cf8895eb8282ecc163fc2802b1dd573ca0241a25/sdv/single_table/copulas.py#L26).